resources/org.ebnf

```diff
@@ -72,4 +72,71 @@ keyword-value = anything-but-newline
 node-property-line = <':'> node-property-name [node-property-plus] <':'> [<' '> node-property-value]
 node-property-name = #"[^\s:+]+"
 node-property-plus = <"+">
-node-property-value = anything-but-newline
\ No newline at end of file
+node-property-value = anything-but-newline
+
+
+(* timestamps
+   https://orgmode.org/worg/dev/org-element-api.html
+
+   The symbol names are carefully chosen as a trade-off between
+   shortness and consistency. They have a prefix, e.g. ts-mod-* for
+   "timestamp modifier", to not get name colissions with other org
+   mode syntax elements.
+
+   We do not try to reproduce the same structure and symbol names as
+   in the org mode spec because
+
+   - it's not possible, e.g. :raw-value cannot be preserved with
+     instaparse (to my understanding)
+   - split up a date or time in its parts does not make much sense in
+     a high-level programming language
+   - e.g. values of :type, :repeater-type, :warning-type cannot be
+     constructed with instaparse (to my understanding)
+
+   Currently not supported: diary format from the Emacs Calendar package
+   https://orgmode.org/manual/Timestamps.html
+
+   *)
+
+
+timestamp = timestamp-active / timestamp-inactive
+
+(* TODO How does that work out: :ts-inner appearing two times in :timestamp-active.
+        And if it works in clojure, does it also work in JS hashes (parse result)? *)
+
+timestamp-active   = <'<'> (ts-inner / ts-inner-span) <'>'> / <'<'> ts-inner <'>--<'> ts-inner <'>'>
+
+timestamp-inactive = <'['> (ts-inner / ts-inner-span) <']'> / <'['> ts-inner <']--['> ts-inner <']'>
+
+(* a single point in time *)
+ts-inner = ts-inner-w-time ts-modifiers / ts-inner-wo-time ts-modifiers
+
+(* a time span *)
+ts-inner-span = ts-inner-w-time <'-'> ts-time ts-modifiers
+
+ts-inner-w-time  = ts-date [<' '+> ts-day] <' '+> ts-time
+ts-inner-wo-time = ts-date [<' '+> ts-day]
+
+ts-date = #"\d{4}-\d{2}-\d{2}"
+
+ts-time = #"\d{1-2}:\d{2}(:\d{2})?"
```

Comment:
@schoettl Amazing job as always on this PR! :pray: :bow: 

Just one question, should this line not be: `ts-time = #"\d{1,2}:\d{2}(:\d{2})?" `

If I make this change, the repl starts and the tests will run, if I don't make the change, the REPL crashes with https://gist.github.com/munen/8f701b56c57490a47f7257b7136a070f

Reply:
Oh right, I fixed this already, but as a magit beginner, I must have accidently discarded this...

Reply:
Now, the all tests should run.